common/src/main/java/com/viaversion/viaversion/protocols/v1_20_3to1_20_5/rewriter/ComponentRewriter1_20_5.java

```diff
@@ -404,7 +404,24 @@ protected CompoundTag convertCanPlaceOn(final AdventureModePredicate value) {
                 convertHolderSet(predicateTag, "blocks", predicate.holderSet());
             }
             if (predicate.propertyMatchers() != null) {
-                final CompoundTag state = convertPredicate(predicate);
+                final CompoundTag state = new CompoundTag();
```

Comment:
Is there any code change in here or why move it up?

Reply:
There was no point in moving it out, it's just a cleanup which I didn't want to make a extra PR for.

Reply:
The high level of nesting makes it less readable than having it in an extra method